model/UnsavedRelationList.php

```diff
@@ -299,7 +299,7 @@ public function avg() {
 		throw new LogicException(__FUNCTION__ . " can't be called on an UnsavedRelationList.");
 	}
 
-	public function byIDs() {
+	public function byIDs(array $ids) {
```

Comment:
If someone's subclassed `UnsavedRelationList` and overloaded `byIDs` and NOT accepted an ID parameter... well... then I don't know what kind of world we are living in.

I suppose I could remove the type hinting if we thought there was a real risk of regression here; personally, I don't